### DISCUSSION TITLE
How to work with audio.

### DISCUSSION THREAD

**Discussion by petrashkevychvasyl** [AxisCommunications/acap-native-sdk-examples#218] [General] [answered]
Hello! I have created app that using vdo_stream.h to stream video to rtms, But without audio. How i can stream with audio?

**Comment by pataxis** [comment, marked as answer]
Hi @petrashkevychvasyl , thanks for you question.

We don't have an audio API in ACAP Native SDK today.

Maybe the recently released [vapix](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/main/vapix) example and the VAPIX [Audio API](https://www.axis.com/vapix-library/subjects/T10100065/section/t10100140/display) could work for you?

**Reply by petrashkevychvasyl** [reply]
Hi @pataxis Thanks for the quick answer!
